docs/stacks-guide-gcp.md

```diff
@@ -0,0 +1,424 @@
+    namespace: gcp
+  reclaimPolicy: Delete
+  routingConfig:
+    routingMode: REGIONAL
+---
+# example-subnetwork defines IP ranges to be used by GKE cluster.
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: Subnetwork
+metadata:
+  name: example-subnetwork
+  namespace: gcp
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: gcp
+  reclaimPolicy: Delete
+  name: example-subnetwork
+  region: us-central1
+  ipCidrRange: "192.168.0.0/24"
+  privateIpGoogleAccess: true
+  secondaryIpRanges:
+    - rangeName: pods
+      ipCidrRange: 10.0.0.0/8
+    - rangeName: services
+      ipCidrRange: 172.16.0.0/16
+  network: projects/$PROJECT_ID/global/networks/example-network
+---
+# example-globaladdress defines the IP range that will be allocated for cloud services connecting
+# to the instances in the given Network.
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: GlobalAddress
+metadata:
+  name: example-globaladdress
+  namespace: gcp
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: gcp
+  reclaimPolicy: Delete
+  name: example-globaladdress
+  purpose: VPC_PEERING
+  addressType: INTERNAL
+  prefixLength: 16
+  network: projects/$PROJECT_ID/global/networks/example-network
+---
+# example-connection is what allows cloud services to use the allocated GlobalAddress for communication. Behind
+# the scenes, it creates a VPC peering to the network that those service instances actually live.
+apiVersion: servicenetworking.gcp.crossplane.io/v1alpha2
+kind: Connection
+metadata:
+  name: example-connection
+  namespace: gcp
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: gcp
+  reclaimPolicy: Delete
+  parent: services/servicenetworking.googleapis.com
+  network: projects/$PROJECT_ID/global/networks/example-network
+  reservedPeeringRanges:
+    - example-globaladdress
+EOF
+
+kubectl apply -f network.yaml
+# Example YAML exists in https://github.com/crossplaneio/crossplane/blob/master/cluster/examples/workloads/kubernetes/wordpress/gcp/network.yaml as well.
+```
+
+For more details about networking and what happens when you run this
+command, see [this document with more details][crossplane-gcp-networking-docs].
+
+It takes a while to create these resources in GCP. The top-level object
+is the `Connection` object; when the `Connection` is ready, everything
+else is too. We can watch it by running (assumes gcp stack is installed in `gcp` namespace):
+
+```
+kubectl -n gcp get connection.servicenetworking.gcp.crossplane.io/example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
+```
+
+## Configure Provider Resources
+
+Once we have the network set up, we also need to tell Crossplane how to
+satisfy WordPress's claims for a database and a Kubernetes cluster.
+The resource classes serve as template for the new claimswe make. 
+The following resource classes allow the GKECluster and CloudSQL claims
```

Comment:
I think this should be "allow the KubernetesCluster and MySQLInstance claims"

Reply:
Should be fixed; I went even more general and didn't us the kind names